<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fsk_1710_config</class>
 <widget class="QDialog" name="fsk_1710_config">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>258</width>
    <height>165</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>241</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLineEdit" name="le_tx_gain"/>
    </item>
    <item>
     <widget class="QLineEdit" name="le_rx_gain"/>
    </item>
    <item>
     <widget class="QLineEdit" name="le_data_size"/>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>tx gain</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>20</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>rx gain</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>20</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>data size</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_save_close">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>120</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>save and close</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
